### PR TITLE
WIP: Enable plugin graphics

### PR DESCRIPTION
**Pull request by rezural**
This is the changes required to upgrade salva to rapier 0.9.

Graphics are not being rendered, this is just so that salva's FluidTestbedPlugin has access to the data it will need.

### CODE REVIEW

File: src_testbed/graphics.rs [hunk 1]
```diff
@@ -392,6 +393,10 @@ impl GraphicsManager {
     pub fn nodes_mut(&mut self) -> impl Iterator<Item = &mut EntityWithGraphics> {
         self.b2sn.values_mut().flat_map(|val| val.iter_mut())
     }
+
+    pub fn prefab_meshes(&self) -> &HashMap<ShapeType, Handle<Mesh>, RandomState> {
+        &self.prefab_meshes
+    }
 }
```
Comment (rezural) [inline src_testbed/graphics.rs:400]: The FluidsTestbedPlugin will presumably need access to the prefab meshes (unless it wants to create its own ones, but this is DRY

File: src_testbed/testbed.rs [hunk 2]
```diff
@@ -864,13 +867,15 @@ fn update_testbed(
...
-    mut gfx_components: Query<(&mut Transform,)>,
+    gfx_components: Query<(&mut Transform,)>,
     mut cameras: Query<(&Camera, &GlobalTransform, &mut OrbitCamera)>,
 ) {
     let meshes = &mut *meshes;
     let materials = &mut *materials;
     let prev_example = state.selected_example;
 
+    let gfx_components = Arc::new(Mutex::new(gfx_components));
+
```
Comment (rezural) [inline src_testbed/testbed.rs:878]: I'm not sure if Arc<Mutex<>> is the way to go here, but passing the mutable transform is not possible without it. Not sure yet if the gfx_components will need to be a mutable query yet..

**Review by rezural — COMMENTED** [review]
There's a couple of println's that snuck in here, I will remove them before this is ready

File: src_testbed/graphics.rs (hunk 1 above)
Comment (rezural) [reply src_testbed/graphics.rs:402]: I think this is no longer needed, I will remove it.

File: src_testbed/testbed.rs (hunk 2 above)
Comment (sebcrozet) [reply src_testbed/testbed.rs:877]: This is odd. I tried removing your `Arc<Mutex<>>` and it compiled fine. Did I miss something?

**Review by sebcrozet — CHANGES_REQUESTED** [review]

**Comment by rezural** [comment]
Probably just me fighting the borrow checker..?

**Comment by rezural** [comment]
OK, I disabled it too, seems to work fine. Do you want me to push up the changes?

**Review by rezural — COMMENTED** [review]
OK, I have updated this.

**Comment by rezural** [comment]
Updated to make collider & rigid body optional

**Review by sebcrozet — APPROVED** [review]

**Comment by sebcrozet** [comment]
Looks great, thanks!

**Pull request merged**